inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/mongodb-cdc/src/main/java/org/apache/inlong/sort/mongodb/DebeziumSourceFunction.java

```diff
@@ -312,6 +312,19 @@ public void snapshotState(FunctionSnapshotContext functionSnapshotContext) throw
             snapshotOffsetState(functionSnapshotContext.getCheckpointId());
             snapshotHistoryRecordsState();
         }
+        if (deserializer instanceof MongoDBConnectorDeserializationSchema) {
+            ((MongoDBConnectorDeserializationSchema) deserializer)
+                    .updateCurrentCheckpointId(functionSnapshotContext.getCheckpointId());
+        }
+    }
+
+    @Override
+    public void notifyCheckpointAborted(long checkpointId) {
```

Comment:
Why flush the audit information when a checkpoint is aborted?

Reply:
Sorry~, it should be add in notifyCheckpointComplete(). thx review ~ 
fix it.